src/components/LiteracyComponents/ResultsComponents/LiteracyTrendsFoundationalChild.tsx

```diff
@@ -0,0 +1,246 @@
+              exclusive
+              onChange={handleView}
+              aria-label="text alignment"
+            >
+              <ToggleButton value="number" aria-label="left aligned">
+                #
+              </ToggleButton>
+              <ToggleButton value="percentage" aria-label="centered">
+                %
+              </ToggleButton>
+            </ToggleButtonGroup>
+          </Grid>
+          <Grid item>
+            <Typography variant="h5" style={{textAlign: "center", fontFamily: 'Arimo'}}>
+              Child Behaviors
+            </Typography>
+          </Grid>
+          <Grid item>
+            <IconButton
+              onClick={handleClick}
+              style={{ boxShadow: '0px 4px 4px rgba(0, 0, 0, 0.25)', padding: 10 }}
+            >
+              <Tooltip title={"Close"} placement={"right"}>
+                <MuiThemeProvider theme={Constants.LiteracyTheme}>
+                  <Badge color='primary' variant='dot' invisible={activityFilter === 0}>
+                    <FilterListIcon />
+                  </Badge>
+                </MuiThemeProvider>
+              </Tooltip>
+            </IconButton>
+            <Menu
+              id='activity-setting'
+              anchorEl={anchorEl}
+              open={Boolean(anchorEl)}
+              elevation={6}
+              getContentAnchorEl={null}
+              keepMounted
+              anchorOrigin={{
+                vertical: 'bottom',
+                horizontal: 'center',
+              }}
+              transformOrigin={{
+                vertical: 'top',
+                horizontal: 'center',
+              }}
+              PaperProps={{
+                style: {
+                  maxHeight: '10em',
+                  // width: '20ch',
+                },
+              }}
+              onClose={(): void => setAnchorEl(null)}
+            >
+              {activitySettings.map((value, index) => {
+                return (
+                  <MenuItem key={index} disabled={index>0 && !childData.some(e => e.activitySetting === value)} onClick={(): void => {handleClose(index)}}>
+                    <ListItemText primary={value} />
+                  </MenuItem>
+                )
+              })}
+            </Menu>
+          </Grid>
+        </Grid>
+      </Grid>
+      <Paper className={classes.root}>
+        <TableContainer className={classes.container}>
+          <Table stickyHeader aria-label="sticky table" size="small">
+            <TableHead>
+              <TableRow>
+                <TableCell style={{minWidth: 250, position: 'sticky', left: 0, backgroundColor: '#fafafa', zIndex: 1}} />
+                {childData.filter(obj => {
+                  return (activityFilter ? obj.activitySetting === activitySettings[activityFilter] : obj)
+                }).map(a => [a.startDate, a.activitySetting]).map((description: Array<string>, index: number) => {
+                  return(
+                    <TableCell
```

Comment:
@thompchr I want the header and footer cells to be hidden when scrolled in this table, but they are still visible. not sure if it is because they are in the TableHead component. any ideas?